packages/next/build/babel/plugins/next-ssg-transform.ts

```diff
@@ -8,9 +8,9 @@ import {
 
 const pageComponentVar = '__NEXT_COMP'
 
-export const EXPORT_NAME_GET_STATIC_PROPS = 'unstable_getStaticProps'
-export const EXPORT_NAME_GET_STATIC_PATHS = 'unstable_getStaticPaths'
-export const EXPORT_NAME_GET_SERVER_PROPS = 'unstable_getServerSideProps'
+export const EXPORT_NAME_GET_STATIC_PROPS = 'getStaticProps'
+export const EXPORT_NAME_GET_STATIC_PATHS = 'getStaticPaths'
+export const EXPORT_NAME_GET_SERVER_PROPS = 'getServerSideProps'
 
 const ssgExports = new Set([
```

Comment:
We need to add all the old names here otherwise build may not succeed.

Reply:
Added